moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -0,0 +1,59 @@
+import BeenhereOutlinedIcon from "@material-ui/icons/BeenhereOutlined";
+import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
+
+export const formatProjectActivity = (change, entityList) => {
+  const entryMap = ProjectActivityLogTableMaps["moped_project"];
+  let changeDescription = "Project updated";
+  let changeIcon = <span className="material-symbols-outlined">summarize</span>;
+  let changeValue = "";
```

Comment:
since these are the "must have" props in your function return (and we're going to repeat the pattern a whole bunch of times), it might self-document better to setup the object up-front:

```javascript
const entry = { 
  changeDescription: "Project updated",
  changeIcon: <span className="material-symbols-outlined">summarize</span>,
  changeValue: "",
}
```